.travis.yml

```diff
@@ -53,11 +54,12 @@ script:
   - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
     make regen_check;
     fi
-  - cargo test --features embed-lgpl-docs
+  - cargo doc --features "dox,embed-lgpl-docs"
+  - cargo test --features "$FEATURES,embed-lgpl-docs"
   # catch any sneaked in lgpl docs
-  - cargo build --features purge-lgpl-docs
-  - cargo doc --features "dox"
```

Comment:
This also still needed

Reply:
Oh, If you checked that `cargo doc --features "dox,embed-lgpl-docs"` adds docs,
then `embed-lgpl-docs` maybe don't needed for test